<launch>

	<param name="/Pioneer3AT/urdf" command="$(find xacro)/xacro.py $(find pioneer3at)/urdf/pioneer3at.xacro" />
   
  <node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher">
    <remap from="robot_description" to="/Pioneer3AT/urdf" />
    <param name="tf_prefix" type="string" value="Pioneer3AT"/>
    <remap from="joint_states" to="/Pioneer3AT/joint_states"/>
    <param name="publish_frequency" type="double" value="20.0"/>
  </node>
  
	<node pkg="pioneer3at" type="joint_state_publisher" name="joint_state_publisher">
	  <param name="tf_future_date" type="double" value="0.05" />
	  
    <remap from="joint_states" to="/Pioneer3AT/joint_states"/>
  </node>
	
</launch>
